tests/pageObjects/AddCountryPage.js

```diff
@@ -0,0 +1,37 @@
+class AddCountryPage {
+    constructor(){
+        this.countryNameFieldSelector = "#country_name";
+        this.countryIsoNameFieldSelector = "#country_iso_name";
+        this.countryIsoFieldSelector = "#country_iso";
+        this.countryIso3FieldSelector = "#country_iso3";
+        this.stateRequiredSelector = "label[for='country_states_required']";
+        this.createCountrySelector = "//button[@type='submit']";
+
+        this.profileIconSelector = "(//*[name()='path'])[8]";
+        this.adminPanelMenuSlector = "//a[normalize-space()='Admin Panel']";
+        this.settingsSpanSelector = "//span[normalize-space()='Settings']";
```

Comment:
`settingsSpanSelector` is not a meaningful name and please avoid using selectors with `normalize`.